modular_skyrat/modules/crafting/crafting_skyrat.dm

```diff
@@ -20,9 +20,9 @@
 	result = /obj/item/gun/energy/beam_rifle
 	reqs = list(/obj/item/gun/energy/laser = 1,
 				/obj/item/assembly/signaler/anomaly/flux = 1,
-				/obj/item/assembly/signaler/anomaly/grav = 1,
```

Comment:
i want it to stay two cores. Had a chance to play more sci and play with the gun